### PR TITLE
SDK package names

### PR DESCRIPTION
**Pull request by wtrocki**
Follow up for email thread: https://groups.google.com/forum/#!topic/aerogear/pNmVTy0saDA

### CODE REVIEW

File: SDK-naming.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Expectations
+
+- Same prefixes standards will be used in the platforms.
+
+## Proposed Solution
+
+### Android:
+
+org.aerogear.ags.android.core - AeroGear Services Core
```
Comment (pb82) [inline SDK-naming.md]: shouldn't this be `org.aerogear.android.ags.core`? (ags and android swapped)

Comment (wtrocki) [reply SDK-naming.md]: My mistake. Swapping that now

**Review by pb82 — APPROVED** [review]
LGTM

File: SDK-naming.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+org.aerogear.android.ags.core - AeroGear Services Core
+
+org.aerogear.android.ags.sync - AeroGear Services Sync
+
+### IOS:
+
+AeroGear/AgsCore
+
+AeroGear/AgsSync etc.
```
Comment (secondsun) [inline SDK-naming.md]: etc could be on a new line and added to the Android section or removed.

File: SDK-naming.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Node/Javascript
+
+@aerogear/ags-core
+
+@aerogear/ags-sync etc.
```
Comment (secondsun) [inline SDK-naming.md]: Same etc comment :)

**Review by secondsun — CHANGES_REQUESTED** [review]

**Review by secondsun — APPROVED** [review]

**Comment by wtrocki** [comment]
@danielpassos - Sadly PR was merged. I will be cool to just create another one with your proposition! Also suggestions for IOS will be good to have.

**Comment by danielpassos** [comment]
@wtrocki We can revisit it in the future before the first release ;)

**Pull request merged**